### ISSUE TITLE
In console.js: "Critical dependency: the request of a dependency is an expression"

### ISSUE THREAD

**Issue by tjr** [baryon/tracer#103]
Getting a warning when using tracer in a VueJS application:

> warning  in ./node_modules/tracer/lib/console.js
> Critical dependency: the request of a dependency is an expression

This seems to be coming from (what is currently) line 105 in console.js:

`userConfig = [require(userConfig[0])];`

because of passing a non-constant expression to require().

**Comment by baryon** [comment]
I think this is a webpack warning

> Note: You can use the ContextReplacementPlugin to modify these values for individual dependencies. This also removes the warning.

set ContextReplacementPlugin of webpack 
```
module: {

  unknownContextRequest: ".",          // disable warning


}

```

**Comment by baryon** [comment]
Fixed on v1.1.0
